tests/test_stamp.py

```diff
@@ -446,7 +448,127 @@ def new_toplevel_only(self, object_types):
     image, _ = galsim.config.BuildStamp(config, obj_num, logger=logger, do_noise=False)
     print(obj_num, image.center, image.array.shape, image.array.sum())
     assert image.array.shape == (32, 32)
-    assert 5 < image.array.sum() < 10  # 8
+    assert 7 < image.array.sum() < 13  # 10
```

Comment:
How about this?  Do you understand why this changed?

Reply:
I think same as above: real QE measurements are better than the baseline.